File: src/fizzbuzz.cs
using System;

public class FizzBuzz {

  public static string FizzBuzz(int n) {
    //ここに素数判定プログラムを実装してください。
    //BEGIN_CHALLENGE
    
    string data;    
    if (n % 15 == 0) data = "FizzBuzz";
    else if(n % 5 == 0) data = "Buzz";
    else if(n % 3 == 0) data = "Fizz";
    else data = n.ToString();

    return data;
    //END_CHALLENGE
  }

}

File: app/fizzbuzz.js
function fizzbuzz(data) {
	//BEGIN_CHALLENGE
	var result;
	if(data % 15 == 0) { result = "FizzBuzz"; }
	else if(data % 5 == 0) {result = "Buzz"; }
	else if(data %3 ==0 ) {result = "Fizz"; }
	else result = data.toString();
	return result;
	//END_CHALLENGE

}

module.exports = fizzbuzz;